### PR TITLE
Use https container

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: .github/workflows/scripts/before_install.sh [hunk 1]
```diff
@@ -85,6 +85,7 @@ fi
 cd pulp-openapi-generator
 sed -i -e 's/localhost:24817/pulp/g' generate.sh
 sed -i -e 's/:24817/pulp/g' generate.sh
+sed -i -e 's/http:/https:/g' generate.sh
```
Comment (mdellweg) [inline .github/workflows/scripts/before_install.sh:88]: Hang on! Can we get that hostname as a parameter for `generate.sh`? This is starting to get weird.

File: .github/workflows/scripts/install.sh [hunk 2]
```diff
@@ -91,6 +91,13 @@ fi
 ansible-playbook build_container.yaml
 ansible-playbook start_container.yaml
 
+# Hack: adding pulp CA to certifi.where()
+sudo docker cp pulp:/etc/pulp/certs/ca.crt /usr/local/share/ca-certificates/pulp_ca.crt
+CERT=$(python -c 'import certifi; print(certifi.where())')
```
Comment (fao89) [inline .github/workflows/scripts/install.sh:96]: This is Mozilla CA bundle, and I'm appending pulp CA to it

File: .github/workflows/scripts/script.sh [hunk 3]
```diff
@@ -79,6 +81,7 @@ if [[ "$TEST" = 'bindings' || "$TEST" = 'publish' ]]; then
   gem build pulp_file_client
   gem install --both ./pulp_file_client-0.gem
   cd ..
+  export SSL_CERT_FILE=/usr/local/share/ca-certificates/pulp_ca.crt
```
Comment (fao89) [inline .github/workflows/scripts/script.sh:84]: but Ruby doesn't use `certifi` lib, so I had to use an OpenSSL env var to point to the CA

**Comment by fao89** [comment]
Blocked by https://github.com/pulp/pulp-oci-images/pull/73 or https://github.com/pulp/pulp-oci-images/pull/84

**Pull request closed without merging**